dev/start.sh

```diff
@@ -0,0 +1,121 @@
+#!/bin/bash
+set -ex
+
+function clean {
+echo 'Removing test environment'
+echo '---'
+docker-compose down -v
+rm -rf inventory.tmp
+}
+
+# normalises project name by filtering non alphanumeric characters and transforming to lowercase
+declare -x COMPOSE_PROJECT_NAME
+
+COMPOSE_PROJECT_NAME=$(echo "${BUILD_TAG:-ansible-plugin-testing}-conjur-host-identity" | sed -e 's/[^[:alnum:]]//g' | tr '[:upper:]' '[:lower:]')
+echo "${COMPOSE_PROJECT_NAME}" > compose_project_name
+
+declare -x ANSIBLE_CONJUR_AUTHN_API_KEY=''
+declare -x CLI_CONJUR_AUTHN_API_KEY=''
+declare cli_cid=''
+declare conjur_cid=''
+declare ansible_cid=''
+
+# get conjur client auth api key
+function api_key_for {
+  local role_id=$1
+  if [ -n "$role_id" ]
+  then
+  docker exec "${conjur_cid}" rails r "print Credentials['${role_id}'].api_key"
+  else
+  echo ERROR: api_key_for called with no argument 1>&2
+  exit 1
+  fi
+  }
+
+# get HFTOKEN
+function hf_token {
+  docker exec "${cli_cid}" bash -c 'conjur hostfactory tokens create --duration-days=5 ansible/ansible-factory | jq -r ".[0].token"'
+  }
+
+function setup_conjur {
+  echo "---- setting up conjur ----"
+  # run policy
+  docker exec "${cli_cid}" conjur policy load root /policy/root.yml
+  # set secret values
+  docker exec "${cli_cid}" bash -ec 'conjur variable values add ansible/target-password target_secret_password'
+  }
+
+# scale up inventory nodes and setup the conjur identity there .
+function setup_conjur_identities {
+  for conjur_identity in config_conjur_identity/*; do
+  teardown_and_setup
+  setup_conjur_identity "$(basename -- "$conjur_identity")"
+  done
+  }
+
+# configure_conjur_identity
+function setup_conjur_identity {
+  echo "---- testing ${conjur_identity} ----"
```

Comment:
Instead of `testing ${conjur_identity}`, this `echo` should be more accurate - maybe `setting up conjur identity on remote nodes`

Reply:
Done.